src/CommandHandlers/JoinRoomHandler.cs

```diff
@@ -15,14 +16,17 @@ public override Task Handle(Client client, NetworkObject receivedObject)
         }
         Room room = Room.GetOrAdd(roomName);
         client.SetRoom(room);
+        if (client.Room == null) return Task.CompletedTask;
```

Comment:
(after call two previous lines) this always should be not null (if I'm not wrong) ... when this is null?